app/services/eventful_service.rb

```diff
@@ -0,0 +1,7 @@
+class EventfulService
+
+  def self.by_zipcode(token, zipcode)
+   response = Faraday.get("http://api.eventful.com/json/events/search?app_key=fQzwFNc4ksLVxn9M&location=#{zipcode}&page_size=16&sort_order=popularity")
```

Comment:
i'd suggest breaking this up a bit to keep it more flexible. you could use the `Faraday.new` syntax then add your parameters later.

something along the lines of this:
```ruby
# per-connection setting
conn = Faraday.new :request => { :params_encoder => Faraday::FlatParamsEncoder }

conn.get do |req|
  # per-request setting:
  # req.options.params_encoder = my_encoder
  req.params['roll'] = ['california', 'philadelphia']
end
# GET 'http://sushi.com?roll=california&roll=philadelphia'
```